Add logos for buildah

rhatdan opened the pull request:
Thanks to Máirín Duffy for building these logos for thBuildah project

Signed-off-by: Daniel J Walsh <dwalsh@redhat.com>

nalind commented:
@TomSweeneyRedHat Oh, I missed that.  The link should probably be [this](https://cdn.rawgit.com/projectatomic/buildah/master/logos/buildah.svg), but it won't be valid until this is merged.

nalind commented on README.md:
```diff
@@ -1,4 +1,6 @@
-[buildah](https://www.youtube.com/embed/YVk5NgSiUw8) - a tool which facilitates building OCI container images
+![buildah logo](https://cdn.rawgit.com/projectatomic/buildah/logos/buildah.svg)
```
I think this needs a `master` between `buildah` and `logos` to point to a branch.

TomSweeneyRedHat replied:
👍

rhatdan commented:
@nalind @TomSweeneyRedHat I fixed the link and fixed a few references to Buildah the project to be capitalized.  

I don't think you will see the logo until this is merged.

TomSweeneyRedHat commented on docs/buildah.md:
```diff
@@ -1,14 +1,14 @@
-## buildah "1" "March 2017" "buildah"
+## Buildah "1" "March 2017" "buildah"
```
nit, should the second buildah be capped?

TomSweeneyRedHat commented:
Travis CI is failing, looks like the RPM build test isn't happy?  The closest thing to an error that I can see is:
#   Running scriptlet: glibc-2.25-10.fc26.x86_64                            93/93BDB1539 Build signature doesn't match environment
# go-compilers-golang-compiler-1-11.fc26.x86_64 was supposed to be installed but is not!

Can't imagine this submit had anything to do with that, maybe a one off?

TomSweeneyRedHat commented:
Hmmm.  Same rpm building error in Red Hat CI.

nalind commented:
bot, retest this please

nalind commented:
@rh-atomic-bot, retest this please

nalind commented:
@rh-atomic-bot, retest this please

nalind commented:
bot, retest this please

TomSweeneyRedHat commented:
Still the same error.  :^(

rhatdan commented:
@cevich @jlebon  any ideas?

cevich commented:
looking

cevich commented:
@rhatdan here's the error:

```
make[1]: Leaving directory '/root/gopath/src/github.com/projectatomic/buildah/docs'
can't load package: package github.com/projectatomic/buildah/vendor/github.com/ostreedev/ostree-go/pkg/otadmin: 
vendor/github.com/ostreedev/ostree-go/pkg/otadmin/admin.go:1:1: expected 'package', found 'EOF'
can't load package: package github.com/projectatomic/buildah/vendor/github.com/ostreedev/ostree-go/pkg/otremote: 
vendor/github.com/ostreedev/ostree-go/pkg/otremote/remoteadd.go:1:1: expected 'package', found 'EOF'
WARNING: deadline exceeded by linter varcheck on . (try increasing --deadline)
WARNING: deadline exceeded by linter varcheck on cmd/buildah (try increasing --deadline)
WARNING: deadline exceeded by linter vet on . (try increasing --deadline)
WARNING: deadline exceeded by linter vet on cmd/buildah (try increasing --deadline)
make: *** [Makefile:50: validate] Error 2
make: Leaving directory '/root/gopath/src/github.com/projectatomic/buildah'
### EXITED WITH CODE 2 AFTER 449s
```

Best guess is some problem with github (happens on occasion).  Though I dunno Go well at all, so maybe a problem w/ the code?

cevich commented:
@rhatdan Aye, looks like a problem with github.  Try going here (I get EOF as well):

https://raw.githubusercontent.com/projectatomic/buildah/master/vendor/github.com/ostreedev/ostree-go/pkg/otadmin/adminupgrade.go

rhatdan commented:
@cgwalters Why do these files exists. They are all empty  should ostree-go drop them?

cgwalters commented:
https://github.com/ostreedev/ostree-go/commit/cb6250d5a6a240b509609915842f763fd87b819d

TomSweeneyRedHat commented:
bot, retest this please

TomSweeneyRedHat commented:
@rh-atomic-bot, retest this please

TomSweeneyRedHat commented:
Not having luck getting the test to kick off.  I'm gonna close and reopen this bad boy, that should do it.

TomSweeneyRedHat commented:
Ok, tests kick off this time.

TomSweeneyRedHat commented:
Yeah Green lights!  @rhatdan or @nalind, let's turn on the logo!
